first/style.css

```diff
@@ -0,0 +1,234 @@
+@import url('https://fonts.googleapis.com/css2?family=Rubik:wght@300;400;500&display=swap'); /* font-family: 'Rubik', sans-serif */
+
+body {
+    font-family: 'Rubik', sans-serif;
+}
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.header{
+    height:600px;
+    background-image: url(./assets/images/5.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    top:0;
```

Comment:
check this https://9elements.com/css-rule-order/
and follow elements order, apply it for all css